configure.ac

```diff
@@ -25,10 +25,17 @@ AC_SUBST([AX_MAJOR_VERSION])
 AC_SUBST([AX_MINOR_VERSION])
 AC_SUBST([AX_POINT_VERSION])
 
+LIBFLUX_HOSTLIST_CURRENT=1
+LIBFLUX_HOSTLIST_REVISION=0
+LIBFLUX_HOSTLIST_AGE=0
+LIBFLUX_HOSTLIST_VERSION_INFO=$LIBFLUX_HOSTLIST_CURRENT:$LIBFLUX_HOSTLIST_REVISION:$LIBFLUX_HOSTLIST_AGE
+AC_SUBST([LIBFLUX_HOSTLIST_VERSION_INFO])
+
```

Comment:
Perhaps in this repo we move the library versioning into the Makefile for each package, instead of in `configure.ac`? Maybe I'm missing some reason this stuff had to be in configure, though.

Reply:
Yeah if there was a reason, I've forgotten.  Seems better to do it in the Makefile.am.